lib/Installation/Registration/RegistrationPage.pm

```diff
@@ -26,6 +26,8 @@ sub init {
     $self->{rdb_skip_registration} = $self->{app}->radiobutton({id => 'skip_registration'});
     $self->{txb_email} = $self->{app}->textbox({id => 'email'});
     $self->{txb_reg_code} = $self->{app}->textbox({id => 'reg_code'});
+    $self->{rdb_rmt_server_registration} = $self->{app}->radiobutton({id => 'register_local'});
```

Comment:
what about `rdb_rmt_server`, it is a bit shorter and rmt is all above registration, should we could omit it.

Reply:
Done!